<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberHistoryPopup</class>
 <widget class="QWidget" name="MemberHistoryPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QLabel" name="member_image">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>191</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_history">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
